deploy-board/deploy_board/templates/swagger-ui/dist/index.html

```diff
@@ -1,123 +1,94 @@
-  <script src='../../../static/swagger-ui-lib/lib/jquery.wiggle.min.js' type='text/javascript'></script>
-  <script src='../../../static/swagger-ui-lib/lib/jquery.ba-bbq.min.js' type='text/javascript'></script>
-  <script src='../../../static/swagger-ui-lib/lib/handlebars-2.0.0.js' type='text/javascript'></script>
-  <script src='../../../static/swagger-ui-lib/lib/underscore-min.js' type='text/javascript'></script>
-  <script src='../../../static/swagger-ui-lib/lib/backbone-min.js' type='text/javascript'></script>
-  <script src='../../../static/swagger-ui-lib/swagger-ui.js' type='text/javascript'></script>
-  <script src='../../../static/swagger-ui-lib/lib/highlight.7.3.pack.js' type='text/javascript'></script>
-  <script src='../../../static/swagger-ui-lib/lib/jsoneditor.min.js' type='text/javascript'></script>
-  <script src='../../../static/swagger-ui-lib/lib/marked.js' type='text/javascript'></script>
-  <script src='../../../static/swagger-ui-lib/lib/swagger-oauth.js' type='text/javascript'></script>
-
-  <!-- Some basic translations -->
-  <!-- <script src='lang/translator.js' type='text/javascript'></script> -->
-  <!-- <script src='lang/ru.js' type='text/javascript'></script> -->
-  <!-- <script src='lang/en.js' type='text/javascript'></script> -->
-
-  <script type="text/javascript">
-    $(function () {
-      var url = window.location.search.match(/url=([^&]+)/);
-      if (url && url.length > 1) {
-        url = decodeURIComponent(url[1]);
-      } else {
-        url = "http://localhost:8080/swagger.json";
-      }
-
-      // Pre load translate...
-      if(window.SwaggerTranslator) {
-        window.SwaggerTranslator.translate();
-      }
-      window.swaggerUi = new SwaggerUi({
-        url: url,
-        dom_id: "swagger-ui-container",
-        supportedSubmitMethods: ['get', 'post', 'put', 'delete', 'patch'],
-        withCredentials: true,
-        onComplete: function(swaggerApi, swaggerUi){
-          if(typeof initOAuth == "function") {
-            initOAuth({
-              clientId: "your-client-id",
-              clientSecret: "your-client-secret-if-required",
-              realm: "your-realms",
-              appName: "your-app-name", 
-              scopeSeparator: ",",
-              additionalQueryStringParams: {}
-            });
-          }
-
-          if(window.SwaggerTranslator) {
-            window.SwaggerTranslator.translate();
-          }
-
-          $('pre code').each(function(i, e) {
-            hljs.highlightBlock(e)
-          });
+    <head>
+        {%load static %}
+        <meta charset="utf-8">
+        <script type="module" src="{% static "js/rapidoc-min.js" %}"></script>
+    </head>
+    <body>
+        <rapi-doc id="apidoc" spec-url="{{ teletraanServiceUrl }}/swagger.json" render-style="read" api-key-name="Authorization" api-key-location="header" api-key-value="token {{ token }}" allow-server-selection="false" allow-authentication="false">
```

Comment:
`setSecurity` is for overriding the default values. The default just works if you try to call Teletraan. 
`setApiKey` won't work because we don't have corresponding securitySchemeId defined in swagger.json